File: RxDemo/src/main/java/com/tamsiree/rxdemo/view/RxFakeAddImageView.java
package com.tamsiree.rxdemo.view;

import android.content.Context;
import android.graphics.PointF;
import android.util.AttributeSet;

import androidx.appcompat.widget.AppCompatImageView;

/**
 * @author tamsiree
 */
public class RxFakeAddImageView extends AppCompatImageView {
    private PointF mPointF;

    public RxFakeAddImageView(Context context) {
        super(context);
    }

    public RxFakeAddImageView(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    public RxFakeAddImageView(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);
    }

    public void setMPointF(PointF pointF) {
        setX(pointF.x);
        setY(pointF.y);
    }

    public PointF getmPointF() {
        return mPointF;
    }

    public void setmPointF(PointF mPointF) {
        setX(mPointF.x);
        setY(mPointF.y);
    }
}

File: RxDemo/src/main/java/com/tamsiree/rxdemo/view/RxFakeAddImageView.kt
package com.tamsiree.rxdemo.view

import android.content.Context
import android.graphics.PointF
import android.util.AttributeSet
import androidx.appcompat.widget.AppCompatImageView

/**
 * @author tamsiree
 */
class RxFakeAddImageView : AppCompatImageView {
    private val mPointF: PointF? = null

    constructor(context: Context?) : super(context)
    constructor(context: Context?, attrs: AttributeSet?) : super(context, attrs)
    constructor(context: Context?, attrs: AttributeSet?, defStyleAttr: Int) : super(context, attrs, defStyleAttr)

    fun setMPointF(pointF: PointF) {
        x = pointF.x
        y = pointF.y
    }

    fun getmPointF(): PointF? {
        return mPointF
    }

    fun setmPointF(mPointF: PointF) {
        x = mPointF.x
        y = mPointF.y
    }
}